addons/spreadsheet/static/tests/charts/model/odoo_chart_plugin.test.js

```diff
@@ -14,6 +14,7 @@ import { insertListInSpreadsheet } from "@spreadsheet/../tests/helpers/list";
 import { createModelWithDataSource } from "@spreadsheet/../tests/helpers/model";
 import { addGlobalFilter } from "@spreadsheet/../tests/helpers/commands";
 import { THIS_YEAR_GLOBAL_FILTER } from "@spreadsheet/../tests/helpers/global_filter";
+import { mockService } from "@web/../tests/web_test_helpers";
```

Comment:
```suggestion
import { makeServerError, mockService } from "@web/../tests/web_test_helpers";
```
From line 1.
This makes the runbot red